include/swift/AST/DiagnosticsSema.def

```diff
@@ -7301,6 +7301,8 @@ ERROR(noimplicitcopy_attr_invalid_in_generic_context,
       none, "'@_noImplicitCopy' attribute cannot be applied to entities in generic contexts", ())
 ERROR(noimplicitcopy_attr_not_allowed_on_moveonlytype,none,
       "'@_noImplicitCopy' has no effect when applied to a noncopyable type", ())
+ERROR(noncopyable_types_cannot_be_resilient, none,
+      "resilient %0 %1 cannot be marked noncopyable yet", (DescriptiveDeclKind, Identifier))
```

Comment:
We don't use the word "resilient" in user facing documentation. How about "public noncopyable type must be `@frozen` in library evolution mode" or something like that, which also lets the developer know what they need to change.

Reply:
Done.